pkg/monitoring/alloy/templates/monitoring-config.yaml.template

```diff
@@ -89,6 +98,8 @@ verticalPodAutoscaler:
       - memory
       - cpu
       controlledValues: "RequestsAndLimits"
+      maxAllowed: 
+        cpu: 500m
       minAllowed:
```

Comment:
This will override the default requests you set up earlier in the values will it not ?

Reply:
(I'm referring to the `minAllowed`)

Reply:
This is the observability opérator not the logging operator

Reply:
That's true yes. Can you create a suggestion for those?